src/examples/DataCenter/Disclaimer.txt

```diff
@@ -0,0 +1,33 @@
+% -------------Software Disclaimer---------------
```

Comment:
there is already the [LICENSE](https://github.com/signetlabdei/qd-realization/blob/master/LICENSE) file. is this needed?

Reply:
Yes! Please do not remove it.

Reply:
Okay I removed this file and included author detail in XML file.